pep-0012.rst

```diff
@@ -50,44 +50,36 @@ To use this template you must first decide whether your PEP is going
 to be an Informational or Standards Track PEP.  Most PEPs are
 Standards Track because they propose a new feature for the Python
 language or standard library.  When in doubt, read PEP 1 for details
-or contact the PEP editors <peps@python.org>.
+or contact a PEP editor for assistance.
 
 Once you've decided which type of PEP yours is going to be, follow the
 directions below.
 
-- Make a copy of this file (``.txt`` or ``.rst`` file, **not** HTML!)
-  and perform the following edits. Prefer ``.rst`` extensions as this
-  will allow PEPs to be rendered on GitHub. ``.txt`` extensions still
-  exists for legacy reasons.
+- Make a copy of this file (the ``.rst`` file, **not** the HTML!) and
+  perform the following edits. The ``.rst`` extension will allow your
+  your PEP to be rendered on GitHub. (``.txt`` extensions still exist
+  for legacy reasons but are discouraged for new PEPs.)
 
-- Replace the "PEP: 12" header with "PEP: 9999" since you don't yet
-  have a PEP number assignment.
+- Replace the "PEP: 12" header with "PEP: 9999" if you don't yet
+  have a PEP number assignment, or with the actual number.
 
 - Change the Title header to the title of your PEP.
 
-- Leave the Version and Last-Modified headers alone; we'll take care
-  of those when we check your PEP into Python's Subversion repository.
-  These headers consist of keywords ("Revision" and "Date" enclosed in
-  "$"-signs) which are automatically expanded by the repository.
-  Please do not edit the expanded date or revision text.
-
 - Change the Author header to include your name, and optionally your
   email address.  Be sure to follow the format carefully: your name
   must appear first, and it must not be contained in parentheses.
   Your email address may appear second (or it can be omitted) and if
   it appears, it must appear in angle brackets.  It is okay to
   obfuscate your email address.
 
-- If none of the authors are Python core developers and this PEP is
-  ready to be submitted to the PEP repository, include a Sponsor header
-  with the name of the core developer sponsoring your PEP.
+- If none of the authors are Python core developers, include a Sponsor
+  header with the name of the core developer sponsoring your PEP.
```

Comment:
Maybe a "if you don't have a sponsor, you're probably not ready to write a PEP" note (with a more friendly wording :slightly_smiling_face:)?

Reply:
I'm hoping that this wording will imply that :) Not sure if we need extra words or not, but here's a first stab at it:

    - If none of the authors are Python core developers, include a Sponsor
      header with the name of the core developer sponsoring your PEP.
      The best place to find a sponsor is the discussion that took place
      prior to the PEP being written - find a core developer who was in
      favour of it, and ask them about sponsoring it.